cpp/tensorrt_llm/batch_manager/cacheFormatter.h

```diff
@@ -116,13 +94,9 @@ class CacheFormatter final : public BaseCacheFormatter
         TLLM_CHECK(mCacheTransBufferManager);
     }
 
-    void formatOutput(LlmRequest const& llmRequest,
-        std::vector<executor::kv_cache::Connection const*> const& connections, CacheState const& selfConfig,
-        SizeType32 selfIdx, CacheState const& destConfig, runtime::BufferManager const& bufferManager) override;
+    void format(TransferSession& session, LlmRequest const& llmRequest) override;
```

Comment:
As previously discussed, the purpose of introducing `Session` is to decouple members from the `LlmRequest` object, thereby addressing the issue of passing the `LlmRequest` directly. Therefore, it's preferable to choose one of the two parameters to pass, rather than maintaining a both-passing intermediate state.